snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -3,41 +3,54 @@
 //
 // * modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc
 // * modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc
-The following `TempoMonolithic` CR creates a TempoMonolithic deployment with trace ingestion over OTLP/gRPC and OTLP/HTTP, storing traces in a supported type of storage and exposing Jaeger UI via a route:
+. Example `TempoMonolithic` CR
 +
 [source,yaml]
 ----
 apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoMonolithic
+kind: TempoMonolithic # <1>
 metadata:
   name: <metadata_name>
   namespace: <project_of_tempomonolithic_instance>
```

Comment:
@rubenvp8510 
Or should this be the namespace?
```suggestion
  namespace: chainsaw-multitenancy
```

Reply:
This is fixed in response to the other comments about the same line in the other (`-tempostack-`) file.